Library/Date+Helpers.swift

```diff
@@ -34,10 +34,10 @@ extension Date {
         return Calendar.autoupdatingCurrent.date(byAdding: .day, value: 1, to: self)!
     }
 
-    func next30days() -> [Date] {
+    func next31days() -> [Date] {
```

Comment:
maybe `andNext30Days` since the array includes the caller? I would personally expect something that's just `next31Days` to be the following 31 days, not including the caller. 

Either that or just document it 😝

Reply:
am not super happy with it, but there's just so much I want to refactor without rewriting the app 👍